install_config/monitoring/modules/accessing-prometheus-alertmanager-grafana.adoc

```diff
@@ -0,0 +1,21 @@
+[id='accessing-prometheus-alertmanager-and-grafana']
```

Comment:
All of your modules need to be in the top-level `modules` folder and contain `-{context}` at the end of the anchor ids.

Reply:
You also need to list the assemblies that use each module in notes at the top of the file.